src/test/perl/lock.t

```diff
@@ -13,38 +13,10 @@ unlink(LOCK_TEST);
 my $lock=CAF::Lock->new(LOCK_TEST);
 
 ok(!$lock->is_locked(), "Unlocked at start");
-my $lockpid=$lock->get_lock_pid();
-
-is($lockpid, undef, "Lock PID undefined on unaquired lock");
 
 ok($lock->set_lock(), "Lock set");
 ok($lock->is_locked(), "Locked on request");
```

Comment:
add a new `CAF::Lock` to try to take the lock on the same file. 
should fail after this (and is an actual test to see if the locking works).